src/Draggable/tests/Draggable.test.js

```diff
@@ -452,7 +452,7 @@ describe('Draggable', () => {
     const callback = jest.fn();
     newInstance.on('drag:move', callback);
 
-    triggerEvent(document, 'mousemove', {
+    triggerEvent(document.body, 'mousemove', {
```

Comment:
@tsov can you help out here?

This test was the only test that started failing after `jest` updates. What I would get is:

<details>
<summary>Jest Error</summary>
<code>
  ● Draggable › triggers `drag:move` drag event on mousedown

    TypeError: Illegal invocation

      at Document.matches (node_modules/jsdom/lib/jsdom/living/generated/Element.js:402:11)
      at conditionFn (src/shared/utils/closest/closest.js:875:32)
      at closest (src/shared/utils/closest/closest.js:921:9)
      at Draggable.value (src/Draggable/Draggable.js:5958:61)
      at invokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:193:27)
      at HTMLUListElementImpl._dispatch (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:111:7)
      at HTMLUListElementImpl.dispatchEvent (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:82:17)
      at HTMLUListElementImpl.dispatchEvent (node_modules/jsdom/lib/jsdom/living/nodes/HTMLElement-impl.js:30:27)
      at HTMLUListElement.dispatchEvent (node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:157:21)
      at MouseSensor.trigger (src/Draggable/Sensors/Sensor/Sensor.js:429:15)
</code>
</details>

I troubleshot this a bit, it eventually seemed like the `Scrollable > hasOverflow` function was being passed a `null` value? On a whim, I changed the test to use `document.body` instead of `document` and the test passed...

What do you think?

Reply:
Jest must have an updated version of jest, I think it's complaining rightfully here, we should be using `document.body` or even `document.documentElement` 👍

Reply:
I'll run through all of these and update to `document.documentElement`

Reply:
I pushed a new commit that changes over a few strangles of `triggerEvent(document,` to use `document.body`. I chose `.body` over `documentElement` as its consistent with the other tests. We can certainly change that in a follow up PR.

For now though, seems all is passing.

Reply:
Sweet, no follow-up needed 👍